frontend/src/features/shell/shell.css

```
@@ -1,27 +1,53 @@
+     account for it in both the `vh` fallback and the `dvh` rule below. */
+  height: calc(100vh - 2 * var(--space-2));
+  margin: var(--space-2);
   overflow: hidden;
   display: grid;
   grid-template-columns:
     var(--nav-width)
     minmax(var(--list-width-min), var(--list-width-max))
     minmax(0, 1fr);
-  background: var(--sidebar);
+  --jv-seam: color-mix(in oklab, var(--border), transparent 60%);
+  border: 1px solid color-mix(in oklab, var(--border), transparent 45%);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)